### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by Lokesh-Krishna**
This is with reference to #308. 
@di72nn This squashes all the commits and also makes all the changes that you mentioned in my earlier pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,71 @@
-[![Build Status](https://travis-ci.org/wallabag/android-app.svg?branch=master)](https://travis-ci.org/wallabag/android-app)
+# wallabag - Android App [![Build Status](https://travis-ci.org/wallabag/android-app.svg?branch=master)](https://travis-ci.org/wallabag/android-app)
 
-# wallabag - Android app
+<img src="unnamed.png" align="left"
+width="200"
+    hspace="10" vspace="10">
```
Comment (Strubbl) [inline README.md]: why have you inserted a newline here?

File: README.md [hunk 2]
```diff
@@ -1,29 +1,71 @@
-[![Build Status](https://travis-ci.org/wallabag/android-app.svg?branch=master)](https://travis-ci.org/wallabag/android-app)
+# wallabag - Android App [![Build Status](https://travis-ci.org/wallabag/android-app.svg?branch=master)](https://travis-ci.org/wallabag/android-app)
 
-# wallabag - Android app
+<img src="unnamed.png" align="left"
```
Comment (Strubbl) [inline README.md]: can we please have a more meaningful name please?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,71 @@
...
+width="200"
+    hspace="10" vspace="10">
 
-wallabag app for Android is a companion app for [wallabag](https://www.wallabag.org).
+wallabag is a self hosted read-it-later app.
```
Comment (Strubbl) [inline README.md]: please add a hyphen between self and hosted --> self-hosted

File: README.md [hunk 4]
```diff
@@ -1,29 +1,71 @@
...
-</a>
-</p>
+    <img alt="Get it on F-Droid"
+        height="80"
+        src="https://f-droid.org/badge/get-it-on.png" />
+ </p>    
+     
+## About
+  
+Wallabag has been made for you to comfortably read and archive your articles.
```
Comment (Strubbl) [inline README.md]: please always write wallabag in small letters, even on beginning of a sentence.

File: README.md [hunk 5]
```diff
@@ -1,29 +1,71 @@
...
+You can download wallabag on wallabag.org and follow the instructions to install it on your own server.
+Alternatively, you can directly sign up on wallabag.it or on Framabag.
```
Comment (Strubbl) [inline README.md]: maybe add links to the strings "wallabag.it"  and "Framabag"?

Comment (Strubbl) [inline README.md]: please change the first "on" to "for" and leave out the second "on"

File: README.md [hunk 6]
```diff
@@ -1,29 +1,71 @@
...
 
+This android application allows you to put a link in your wallabag instance, letting you can read your wallabag links offline.
 
-## Contributing
+This application was originally created by Jonathan GAULUPEAU and released under the GNU GPLv3 licence.
+wallabag is a creation from Nicolas Lœuillet released under the MIT licence.
+
+## Features  
+The android app lets you   
+- Connect to your self hosted wallabag instance or connect to your wallabag.it account or Framabag account.
```
Comment (Strubbl) [inline README.md]: pls add the hyphen to self-hosted

File: README.md [hunk 7]
```diff
@@ -1,29 +1,71 @@
...
+This android application allows you to put a link in your wallabag instance, letting you can read your wallabag links offline.
 
-## Contributing
+This application was originally created by Jonathan GAULUPEAU and released under the GNU GPLv3 licence.
+wallabag is a creation from Nicolas Lœuillet released under the MIT licence.
+
+## Features  
+The android app lets you   
+- Connect to your self hosted wallabag instance or connect to your wallabag.it account or Framabag account. 
+- Supports wallabag 2.0
```
Comment (Strubbl) [inline README.md]: we are already at release 2.1.5. So maybe we add a string here "and newer" or the like?

File: README.md [hunk 8]
```diff
@@ -1,29 +1,71 @@
...
+Alternatively, you can directly sign up on wallabag.it or on Framabag.
 
+This android application allows you to put a link in your wallabag instance, letting you can read your wallabag links offline.
 
-## Contributing
+This application was originally created by Jonathan GAULUPEAU and released under the GNU GPLv3 licence.
+wallabag is a creation from Nicolas Lœuillet released under the MIT licence.
+
+## Features  
+The android app lets you
```
Comment (Strubbl) [inline README.md]: in the following listing please be consistent with the dots at the end of each point. either let all end with a dot or let no point end with a dot

File: README.md [hunk 9]
```diff
@@ -1,29 +1,71 @@
...
+## Features  
+The android app lets you   
+- Connect to your self hosted wallabag instance or connect to your wallabag.it account or Framabag account. 
+- Supports wallabag 2.0
+- Completely ad-free
+- Increase and decrease the size of the font and also switch between a serif or sans-serif font for a more comfortable reading experience.
+- Switch between numerous themes.  
+- Needs no special permissions on Android 6.0+
+
+There are a lot more feautures that we are working on currently and we are looking forward to bringing them to you, including complete support for wallabag 2.0 and a redesign of the app to bring it even more in alignment with Google's Material Design guidelines.
```
Comment (Strubbl) [inline README.md]: feautures --> features

the 2.0 is confusing here. maybe leave out the ".0"?

File: README.md [hunk 10]
```diff
@@ -1,29 +1,71 @@
...
+width="200"
+    hspace="10" vspace="10">](Wallabag%20Reading%20List.png)
+[<img src="Wallabag%20Article%20View.png" align="center"
+width="200"
+    hspace="10" vspace="10">](Wallabag%20Article%20View.png)
 
-wallabag app is a free and open source project developed by volunteers. Any contributions are welcome. Here's a few ways you can help:
+## Permissions
+
+On Android versions prior to Android 6.0, wallabag required the following:
```
Comment (Strubbl) [inline README.md]: required --> requires

File: README.md [hunk 11]
```diff
@@ -1,29 +1,71 @@
...
+- Full Network Access
+- View Network Connections
+- Run at startup  
+
+The "Run at startup" permission was only used if Auto-Sync was enabled and was not utilised when Auto-Sync was disabled. The other two permissions were made use of for downloading content for viewing offline.
```
Comment (Strubbl) [inline README.md]: Please use present instead of past tense.

File: README.md (hunk 5 above)
Comment (Lokesh-Krishna) [inline README.md]: I think the first "on" can be better replaced with a "from" and that's what I have done. I don't think "for" would be the right word there.
Also, isn't the second "on" something that should be there for the sentence to make sense?

**Comment by Lokesh-Krishna** [comment]
@Strubbl I've made all the changes. Thank you so much for the detailed review :)

File: README.md (hunk 5 above)
Comment (Lokesh-Krishna) [inline README.md]: Adding links to those two strings strangely converts all text preceding it into a link to the F-Droid landing page for wallabag.
I know this is more likely to be me not paying attention to something but I couldn't figure it out and thought it was safer to drop the links altogether. I have added them now and am facing the same problem.

File: README.md [hunk 12]
```diff
@@ -1,29 +1,70 @@
...
+        src="https://f-droid.org/badge/get-it-on.png" />
+ </p>    
+     
+## About
+  
+wallabag has been made for you to comfortably read and archive your articles.
+You can download wallabag from wallabag.org and follow the instructions to install it on your own server.
+Alternatively, you can directly sign up for [wallabag.it](https://www.wallabag.it/en) or [Framabag](https://www.framabag.org/).
 
+This android application allows you to put a link in your wallabag instance, letting you can read your wallabag links offline.
```
Comment (Strubbl) [inline README.md]: in the second part of the sentence the word "can" is too much

Comment (Lokesh-Krishna) [inline README.md]: Ah, I completely missed that. Thanks :)

File: README.md (hunk 8 above)
Comment (Lokesh-Krishna) [inline README.md]: I think I have taken care of this now?

Comment (Strubbl) [inline README.md]: Yes. I made the comment to the wrong line, so it wasn't updated by your most recent changes.

Comment (Lokesh-Krishna) [inline README.md]: Is there anything I should do on my side then?

**Comment by Strubbl** [comment]
Also, the person which is going to merge this PR, please selects the option "squash and merge" instead of just "merge pull request". This option is available at the arrow down placed next to the merge button.

File: README.md (hunk 8 above)
Comment (Strubbl) [inline README.md]: No, everything is fine. Thanks, the bullet points are consistent now

**Pull request merged**